New feature to display simulation outputs on model diagrams (#482)
Metroscope-dev/metroscope-modeling-library: CHANGELOG.md

## [Unreleased]

### ✨ Added <!--Make sure to add a link to the PR and issues related to your change-->

> - Added a deature to display simulation outputs on the models' diagrams [#482](https://github.com/Metroscope-dev/metroscope-modeling-library/pull/482)

Reviewer: ```suggestion
- New feature to display simulation outputs on model diagrams [#482](https://github.com/Metroscope-dev/metroscope-modeling-library/pull/482)
```
